### PR TITLE
feat: load cart and add item to it

### PR DESCRIPTION
**Pull request by zoltantakacs-born**
Composables to create the cart, load back if the previous cartId is available and add item to it. Unit tests and types included.

## Related Issue
VBC-148
VBC-153

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: packages/composables/src/useCart/actions/load.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  if (!cartId) {
+    const { data } = await context.$bigcommerce.api.createCart({
+      data: {
+        line_items: []
+      }
+    });
+
+    context.$bigcommerce.config.app.$cookies.set(COOKIE_KEY_CART_ID, data.id, {
+      path: '/',
+      maxAge: 60 * 60 * 24 * 7
```
Comment (scernisevs) [inline packages/composables/src/useCart/actions/load.ts:27]: Hi @zoltantakacs-born , 

The PR looks great! small Q:
Could we move the value of cookie's maxAge into a constant file or themeConfig?
I imagine we might reuse the same maxAge value into other cookies that we might have to create in the future

**Review by scernisevs — APPROVED** [review]
LGTM! :) 1 minor comment / request but everything else looks awesome

File: packages/composables/src/useCart/actions/load.ts (hunk 1 above)
Comment (zoltantakacs-born) [reply packages/composables/src/useCart/actions/load.ts:27]: of course, I wanted to do that, just forgot it 😄

**Review by WojtekTheWebDev — APPROVED** [review]
LGTM!

**Pull request merged**